R/primer-validation-functions.R

```diff
@@ -93,10 +93,12 @@ primer_filtering <- function(file, Ref, logfile, SampleSize=100000, setwd=NULL){
 #' @param OTUsizeFilter. Required. Size of the OTUs to filter out (integer).
 #' 
 #' @import ggplot2
+#' @import phyloseq
+#' @import purrr
+#' @import dplyr
+#' @import ggtree
+#' @import gridExtra
 #' @importFrom Biostrings writeXStringSet
```

Comment:
Hey christophe.  Do we need to import the full package or can we use only particular functions? @importFrom allows you to limit the scope of what is imported and is generally preferable to @import unless you are using a wide range of functionality. You would know best for this piece of code but I wanted to be sure of your intent before merging.

zcp